hw1/1.1/1.1/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace _1._1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enter the size of the array:");
+            int length = Convert.ToInt32(Console.ReadLine());
+            int[] array = new int[length];
```

Comment:
```suggestion
            var array = new int[length];
```
Из правой части очевиден тип переменной